src/taipy/_cli/_scaffold_cli.py

```diff
@@ -40,5 +40,6 @@ def parse_arguments(cls):
         args = _CLI._parse()
 
         if getattr(args, "which", None) == "create":
-            cookiecutter(cls._TEMPLATE_MAP[args.template])
+            application_path = cookiecutter(cls._TEMPLATE_MAP[args.template])
+            cls.__logger.info(f"Taipy application created at {application_path}")
```

Comment:
I would not use the taipy logger here since it is designed for runtime when a Taipy application is running. 

Here we are at development time. It is configured to display the level, the timestamp, and maybe morethings that are useless for a command line.
Probably a print is enough.

What do you think?

Reply:
In my defense, Taipy logger looks better 🤣

Reply:
🤣

Reply:
Let's wait for Fabien review. I don't have strong opinion about this.

Reply:
Just to complete my comment: 
Logger (and so Taipy logger) is made for logs (big information 🤣 ). It is configurable to display more or less information on logs, so it is easier to automate a post-process, a search, or whatever treatment in the log base.

An output command line is not a log (or not in the same sense at least).
You don't have post-treatments on command line outputs. And usually, we don't want to make the command line output configurable. Making the command line output look like a log makes it strange to me.

Anyway, I won't fight on it. It's up to you.

Reply:
I do understand Jean-Robin's reluctance to use the internal Taipy logger for this "application". That indeed was not the purpose of this class.
And that would have been my first recommendation as well.

Now, I can see that no matter what, the 'taipy' script and the 'create' command, in particular, do depend on the Taipy library anyway (contrary to "light" scripts that would be more or less independent of deeply internal features of libraries they depend on).
So I'm good with using the Taipy logger. If Giang thinks the output is "better", and also more in line with other messages that Taipy would produce, in other contexts, let's stick with it.